assets/scss/modules/_phase-banner.scss

```diff
@@ -0,0 +1,8 @@
+.header-banner-wrapper {
+    max-width: 960px;
+    margin: 0 auto;    
+    display: block;
+    @media screen and (max-width: 986px) {
```

Comment:
None of this code needs writing. I think you can just add the `.header-banner-wrapper` selector to `#content` [here](https://github.com/hmrc/assets-frontend/blob/351d770835d1f9eb9ce17953e18e6bc084083a7a/assets/scss/base/_layouts.scss#L13) to adopt the same styles